.github/workflows/fossa.yml

```diff
@@ -8,20 +8,25 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Use Go ^1.14.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.14.x
+
+      - name: Add GOPATH to GITHUB_ENV
+        run: echo "GOPATH=$(go env GOPATH)" >>"$GITHUB_ENV"
+
+      - name: Add GOPATH to GITHUB_PATH
+        run: echo "$GOPATH/bin" >>"$GITHUB_PATH"
```

Comment:
? 

Do these GITHUB_ENV/GITHUB_PATH vars contain file names?

Reply:
Is this probably what you are looking for?
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#environment-files

Reply:
It's also not clear to me why this is needed. Perhaps leave a comment in the code?

Reply:
Well, all it does is expose the GOPATH environment variable and add GOPATH to the PATH. Of course, we can do it more typically. Or are you asking why these are necessary? If so, it's necessary because fossa-cli needs them when it analyzes.